www/docs/en/dev/guide/cli/index.md

```diff
@@ -133,86 +120,78 @@ Android target: installed android-19,android-21,android-22,android-23,Google Inc
-execute a device image, so that you can launch the app from the home
-screen and see how it interacts with many platform features.  Run a
-command such as the following to rebuild the app and view it within a
-specific platform's emulator:
+SDKs for mobile platforms often come bundled with emulators that execute a device image, so that you can launch the app from the home screen and see how it interacts with many platform features.  Run a command such as the following to rebuild the app and view it within a specific platform's emulator:
 
 ```bash
-$ cordova emulate android
+cordova emulate android
 ```
 
 ![]({{ site.baseurl }}/static/img/guide/cli/android_emulate_init.png)
 
-Following up with the `cordova emulate` command refreshes the emulator
-image to display the latest application, which is now available for
-launch from the home screen:
+Following up with the `cordova emulate` command refreshes the emulator image to display the latest application, which is now available for launch from the home screen:
 
 ![]({{ site.baseurl }}/static/img/guide/cli/android_emulate_install.png)
 
-Alternately, you can plug the handset into your computer and test the
-app directly:
+Alternately, you can plug the handset into your computer and test the app directly:
 
 ```bash
-$ cordova run android
+cordova run android
 ```
 
-Before running this command, you need to set up the device for
-testing, following procedures that vary for each platform.
+Before running this command, you need to set up the device for testing, following procedures that vary for each platform.
+
+**See Also:**
 
-### See Also
 - [Setting up Android emulator](../../guide/platforms/android/index.html#setting-up-an-emulator)
 - [Cordova run command reference documentation][cdv_run]
 - [Cordova emulate command reference documentation][cdv_emulate]
 
 ## Add Plugins
 
-You can modify the default generated app to take advantage of standard web technologies,
-but for the app to access device-level features, you need to add plugins.
+You can modify the default generated app to take advantage of standard web technologies, but for the app to access device-level features, you need to add plugins.
 
-A _plugin_ exposes a Javascript API for native SDK functionality. Plugins are typically hosted on
-npm and you can search for them on the [plugin search page](/plugins/). Some key APIs are provided by the Apache Cordova open source project and these are referred to as [Core Plugin APIs]. You can also use the CLI to launch the search page:
+A _plugin_ exposes a Javascript API for native SDK functionality. Plugins are typically hosted on npm and you can search for them on the [plugin search page](/plugins/). Some key APIs are provided by the Apache Cordova open source project and these are referred to as [Core Plugin APIs]. You can also use the CLI to launch the search page:
 
 ```bash
-$ cordova plugin search camera
+cordova plugin search camera
```

Comment:
I don't think this feature exists anymore.

1. the command requires to be inside a cordova project

2. this is what it does for me:

```
cordova plugin search cordova-plugin-camera
@totalpave/cdv-iri-log 0.0.0 "cdv-iri-log"
@totalpave/cordova-plugin-device-motion 1.0.2 "Device Motion"
@totalpave/cordova-plugin-file 1.0.1 "File"
@totalpave/cordova-plugin-geolocation 1.0.1 "Geolocation"
@totalpave/cordova-plugin-insets 0.1.6 "cordova-plugin-insets"
@totalpave/cordova-plugin-libcxx 25.0.8775105 "libc++_shared.so"
@totalpave/cordova-plugin-libsqlite 0.0.16 "SQLite Binaries v3.39.2"
@totalpave/cordova-plugin-libtilegen 0.0.6 "libtilegen"
@totalpave/cordova-plugin-sqlite 0.0.1 "SQLite 3.39.2"
cordova-plugin-androidx-adapter 1.1.3 "cordova-plugin-androidx-adapter"
cordova-plugin-buildinfo 4.0.0 "BuildInfo"
cordova-plugin-crosswalk-data-migration 1.0.0 "Crosswalk Data Migration Plugin"
cordova-plugin-device 2.1.0 "Device"
cordova-plugin-dialogs 2.0.2 "Notification"
cordova-plugin-disable-ios11-statusbar 1.0.0 "cordova-plugin-disable-ios11-statusbar"
cordova-plugin-googlemaps 3.0.4 "cordova-plugin-googlemaps"
cordova-plugin-insomnia 4.3.0 "Insomnia (prevent screen sleep)"
cordova-plugin-ionic-webview 5.0.0 "cordova-plugin-ionic-webview"
cordova-plugin-localization-strings 4.2.0 "Localization"
cordova-plugin-migrate-localstorage 0.0.3 "Migrate WebView Data"
cordova-plugin-screen-orientation 3.0.2 "Screen Orientation"
cordova-plugin-statusbar 1.3.1 "StatusBar"
cordova.plugins.diagnostic 6.1.0 "Diagnostic"
es6-promise-plugin 4.2.2 "Promise"
```

Reply:
Removed.